cloud/account.go

```diff
@@ -16,6 +16,15 @@ import (
 	"google.golang.org/protobuf/types/known/timestamppb"
 )
 
+type AuthMethod string
+
+const (
+	SSHAuthMethod       AuthMethod = "ssh"
+	PasswordAuthMethod  AuthMethod = "password"
+	TokenAuthMethod     AuthMethod = "token"
+	CachedJWTAuthMethod AuthMethod = "cached jwt"
```

Comment:
super minor nit: I believe that idiomatic go puts the type first, e.g. [http methods](https://pkg.go.dev/net/http#pkg-constants). `AuthMethodSSH` and such are probably more idiomatic.